gradle/verification-metadata.xml

```diff
@@ -2667,6 +2667,12 @@
             <sha256 value="c81404e11744523817c1b560aa84c9c44ccfc32f62916d39f7763767f8c3db94" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="io.github.java-diff-utils" name="java-diff-utils" version="4.10">
```

Comment:
I wonder this addition is the only relevant thing you need to add?

Reply:
I only need the 3 dependencies in `build.gradle.kts`. I ran the following command to generate it: `./gradlew --write-verification-metadata md5,sha256 help` and I'm not sure why it added a bunch more dependencies. But I'll revert the file and add the 3 dependencies manually.